javalib/src/main/scala/java/util/concurrent/atomic/AtomicReferenceArray.scala

```diff
@@ -0,0 +1,43 @@
+package java.util.concurrent.atomic
+
+class AtomicReferenceArray[E <: AnyRef](
+    length: Int) extends Serializable {
```

Comment:
`length` is redundant, you can get it from `inner.length`. But you would have to change the way `inner` is initialised.